Directory.Build.props

```diff
@@ -86,6 +86,7 @@
       $(LocalAppData)\Microsoft\Windows\INetCache\**;
       A:\;
```

Comment:
should these other ones be `\**` as well?

Reply:
For whatever reason, the VS telemetry stuff only hits the root of these non-existing drives.

Reply:
Example from recent logs:

```
7768, GENERIC_WRITE, 8192, Write, CreateFile, C:\Users\cloudtest\AppData\Local\Microsoft\Windows\INetCache\IE\dyntelconfig[1].cache, 0
7768, GENERIC_WRITE, GENERIC_READ, FILE_FLAG_OPEN_NO_RECALL, FILE_FLAG_OVERLAPPED, ReadWrite, CreateFile, C:\Users\cloudtest\AppData\Local\Microsoft\Windows\INetCache\IE\dyntelconfig[1].cache, 183
7768, DELETE, FILE_FLAG_DELETE_ON_CLOSE, Write, DeleteFile, C:\Users\cloudtest\AppData\Local\Microsoft\Windows\INetCache\IE\dyntelconfig[1].cache, 0
7768, GENERIC_READ, FILE_FLAG_OPEN_NO_RECALL, FILE_FLAG_SEQUENTIAL_SCAN, Probe, GetFileAttributesEx, A:\, 21
7768, GENERIC_READ, FILE_FLAG_OPEN_NO_RECALL, FILE_FLAG_SEQUENTIAL_SCAN, Probe, GetFileAttributesEx, E:\, 21
7768, GENERIC_READ, FILE_FLAG_OPEN_NO_RECALL, FILE_FLAG_SEQUENTIAL_SCAN, Probe, GetFileAttributesEx, A:\, 21
```

Pid 7768 was VCTIP.exe for reference.